From d4a91b5d422bbbd1b4519ea1016e47a83c41d9d7 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 09:56:45 +0800
Subject: [PATCH] Test case3-E for Delete an default value FeatureFlag PCD

Description: Delete the FeatureFlag PCD with the same value ("gEfiMdeModulePkgTokenSpaceGuid.PcdPeiFullPcdDatabaseEnable|TRUE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..0143e1a52b 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -177,7 +177,7 @@
 ################################################################################
 [PcdsFeatureFlag]
 gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|TRUE
-gEfiMdeModulePkgTokenSpaceGuid.PcdPeiFullPcdDatabaseEnable|TRUE
+#gEfiMdeModulePkgTokenSpaceGuid.PcdPeiFullPcdDatabaseEnableTRUE
 
 
 [PcdsFixedAtBuild]
-- 
2.27.0.windows.1

